docs/tools/aiven-operator.rst

```diff
@@ -0,0 +1,217 @@
+Aiven Operator for Kubernetes
+=============================
+
+    Kubernetes, also known as K8s, is an open-source system for automating deployment, scaling, and management of containerized applications. Kubernetes Operators are software extensions to Kubernetes that make use of custom resources to manage applications and their components. `Kubernetes website <https://kubernetes.io/>`_.
```

Comment:
here (and elsewhere), we have "Operator" with a capital O, but it isn't a proper noun other than in the title of the project "Aiven Operator". The same goes for other English words here, such as Pod, Secret, etc. They do have special meanings in Kubernetes , but writing them in an unexpected way doesn't make the text easier to read.

Reply:
Fixed.

Just stating here: IMHO, Kubernetes-specific resources (like Pods and Secrets) should be capitalized when talking about Kubernetes heheh